spec/httpi/httpi_spec.rb

```diff
@@ -294,7 +292,7 @@
       end
 
       HTTPI::Adapter::ADAPTERS.each do |adapter, opts|
-        unless (adapter == :em_http && RUBY_VERSION =~ /1\.8/) || (adapter == :curb && RUBY_PLATFORM =~ /java/)
+        unless (adapter == :em_http || adapter == :curb) && RUBY_PLATFORM =~ /java/
```

Comment:
As mentioned, a platform-to-adapters-supported-on-that-platform could be interesting.

Reply:
Yeah this was a quick hack to get it passing since it's already 1AM lol.

Reply:
All in due time, that's a note for future PRs.

Reply:
Yes

Reply:
Yes